src/OpenTelemetry/Trace/ActivityProcessor.cs

```diff
@@ -167,7 +167,9 @@ protected virtual bool OnForceFlush(int timeoutMilliseconds)
         /// wait indefinitely.
         /// </param>
         /// <remarks>
-        /// This function should not throw exception.
+        /// This function is called synchronously on the thread which made the
+        /// first call to <c>Shutdown</c>. This function should not throw
+        /// exception.
```

Comment:
nit: Same comment about `should not throw exception` on a few spots. It looks like OnStart/OnEnd use "exceptions" plural, which seems fine.